.github/workflows/rust.yml

```diff
@@ -100,32 +100,32 @@ jobs:
 
     - name: Build (wasm32-unknown-unknown)
       env:
-        RUSTFLAGS: -D warnings -C link-args=-S
```

Comment:
To confirm, the reason `-S` is removed from linker arguments is that we are now relying on debug symbol stripping by rustc, and also linker `-S` was ineffective in removing debug symbols?

Reply:
`-C link-args=-S` was effective, but it was running only as part of the CI.

Reply:
Got it, thanks for the explanation.